Contributors.md

```diff
@@ -0,0 +1,2 @@
+# Contributors
```

Comment:
Can you remove this, I think it shows automatically in github which are the contributors, and I don't want to maintain a list :)

Reply:
Ok, removed, I just followed the newbie instructions you sent. :)